cou/steps/backup.py

```diff
@@ -25,7 +25,7 @@
 )
 
 
-def backup() -> None:
+def backup() -> AnyStr:
```

Comment:
`os.path.abspath` return string, no bytes so `str` is enough.

Reply:
AnyStr in posixpath.pyi

Reply:
I do not understand? I have no idea what is `posixpath.pyi`